sdk/identity/identity/src/msal/nodeFlows/msalClient.ts

```diff
@@ -270,7 +269,10 @@ export function generateMsalConfiguration(
   );
 
   // TODO: move and reuse getIdentityClientAuthorityHost
-  const authority = getAuthority(resolvedTenant, getAuthorityHost(msalClientOptions));
+  const authority = getAuthority(
```

Comment:
Was this intentional? I believe we need to use `getAuthorityHost`

Reply:
Accidental